blocks/small-promo-block/features/small-promo/default.jsx

```diff
@@ -90,7 +90,7 @@ SmallPromo.propTypes = {
         above: 'Image Above',
         below: 'Image Below',
       },
-    }),
+    }).isRequired,
```

Comment:
does arc proptypes use `isRequired` -- haven't seen this before

Reply:
don't think you need to include it

Reply:
Poorva encountered the drop down with empty "--", which the default should be "Image Right".

Reply:
huh, so you're saying `isRequired` is needed to have the default render?